inst/source_template/README.Rmd

```diff
@@ -0,0 +1,63 @@
+---
+output: github_document
+---
+
+<!-- README.md is generated from README.Rmd. Please edit that file -->
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>",
+  fig.path = "man/figures/README-",
+  out.width = "100%"
+)
+```
+
+# {{{ Package }}}
+
+<!-- badges: start -->
+[![Project Status: Concept – Minimal or no implementation has been done yet, or the repository is only intended to be a limited example, demo, or proof-of-concept.](https://www.repostatus.org/badges/latest/concept.svg)](https://www.repostatus.org/#concept)
+![GitHub](https://img.shields.io/github/license/inbo/{{{ Package }}})
+[![R build status](https://github.com/inbo/{{{ Package }}}/workflows/R-CMD-check/badge.svg)](https://github.com/inbo/{{{ Package }}}/actions)
+[![Codecov test coverage](https://codecov.io/gh/inbo/{{{ Package }}}/branch/master/graph/badge.svg)](https://codecov.io/gh/inbo/{{{ Package }}}?branch=master)
+![GitHub code size in bytes](https://img.shields.io/github/languages/code-size/inbo/{{{ Package }}}.svg)
+![GitHub repo size](https://img.shields.io/github/repo-size/inbo/{{{ Package }}}.svg)
+<!-- badges: end -->
+
+The goal of {{{ Package }}} is to ...
+
+## Installation
+
+You can install the development version from [GitHub](https://github.com/) with:
+
+``` r
+# install.packages("remotes")
+remotes::install_github("inbo/{{{ Package }}}")
+```
+
+## Example
+
+This is a basic example which shows you how to solve a common problem:
+
+```{r example}
+library({{{ Package }}})
+## basic example code
+```
+
+What is special about using `README.Rmd` instead of just `README.md`? 
+You can include R chunks like so:
+
+```{r cars}
+summary(cars)
+```
+
+You'll still need to render `README.Rmd` regularly, to keep `README.md` up-to-date.
```

Comment:
Is there a reason why this is not rendered automatically using Github Actions?  (I'm just lazy, and above all, I don't want to check this all the time if others contribute to my packages...)

Reply:
`check_documentation()` renders the Rmd during the GHA. Adding the rendered version would require a new or amended commit.

Reply:
So the GHA warns if the rendered version has to be added?

Reply:
The checks fail when you forgot the render the README.